live-examples/css-examples/pseudo-class/not.html

```diff
@@ -0,0 +1,9 @@
+<p>
+    <b>Марс</b> – одна з найбільш подібних до Землі планет. День <b>Марсу</b> – майже такий самий, як день Землі, лишень
```

Comment:
Нетипове формулювання, давай краще так
```suggestion
    <b>Марс</b> – одна з найбільш подібних до Землі планет. День на <b>Марсі</b> – майже такий самий, як на Землі, лишень
```